src/utils.ts

```diff
@@ -131,6 +133,25 @@ export class Utils {
         return serverTokens;
     }
 
+    public static configServerCredentials(): string[] | undefined {
+        let url: string | undefined = process.env.JF_URL;
+        let user: string | undefined = process.env.JF_USER;
+        let password: string | undefined = process.env.JF_PASSWORD;
+        let accessToken: string | undefined = process.env.JF_ACCESS_TOKEN;
+
+        if (url) {
+            let configCmd: string[] = ['c', 'add', Utils.DEFAULT_SERVER_ID, '--url', url, '--interactive=false'];
+
+            if (user && password) {
+                configCmd.push('--user', user, '--password', password)
+                return configCmd
+            } else if (accessToken) {
+                configCmd.push('--access-token', accessToken)
+                return configCmd
+            }
+        }
```

Comment:
Please consider to following:
```suggestion
        if (url) {
            let configCmd: string[] = ['c', 'add', Utils.DEFAULT_SERVER_ID, '--url', url];
            if (accessToken) {
                configCmd.push('--access-token', accessToken);
            } else if (user && password) {
                configCmd.push('--user', user, '--password', password);
            }
            return configCmd;
        }
```
Reasons:
1. Allow using Artifactory servers without credentials (URL only)
2. Remove redundant interactive flag because CI=true
3. Give first priority to access tokens